portafly/src/i18n/locales/en/product/edit_product.yml

```diff
@@ -0,0 +1,25 @@
+---
+# ------------------ Product > Edit product
+# Page details
+
+page_title: "Product" #--- PROPOSED
```

Comment:
This is a doubt: do we still want to remark that `product` and `backend`are APIs? 
If 3scale is an API Management solution, I think that adding the term `API` to products and backends is redundant.
WDYT @cntlsn ?

Reply:
@porueesq it feels like this is a topic that should be discussed with the product team because it probably impacts other areas of the product as well, and we should be consistent.
My two cents is that remarking that products and backends are APIs could help UX, mostly on day one.